packages/trader/src/AppV2/Hooks/useOrderDetails.ts

```diff
@@ -22,7 +22,7 @@ import { isCancellationExpired } from 'Stores/Modules/Trading/Helpers/logic';
 import { Localize } from '@deriv/translations';
 import React from 'react';
 
-const CARD_LABELS = getCardLabelsV2();
```

Comment:
The localized values from `CARD_LABELS` were not updating on language change, because function `getCardLabelsV2` should be called not here, but inside of `useOrderDetails` func